### PR TITLE
OCT-299 Implement ORCID branding requirements

### PR DESCRIPTION
**Pull request by florin-holhos**
The purpose of this PR was to add ORCID logo icon 
image

and to replace ORCiD occurances with ORCID when we're mentioning the organisation and ORCID iD where we mention the ID of an ORCID user

---

### Acceptance Criteria:

As per [OCT-299](https://jiscdev.atlassian.net/browse/OCT-299)

### Tests:

---

### Screenshots:
image
image
image
image

### CODE REVIEW

**Review by Harry-Jisc — DISMISSED** [review]

File: ui/src/components/Publication/Card/index.tsx [hunk 1]
```diff
@@ -23,7 +24,22 @@ const Card: React.FC<Props> = (props): React.ReactElement => (
                     : props.publication.title}
             </p>
             <span className="mb-4 block font-montserrat text-sm text-grey-800 transition-colors duration-500 dark:text-white-50">
-                {props.publication.user.firstName[0]}. {props.publication.user.lastName}
+                {props.publication.user.id === 'octopus' ? (
+                    <>
+                        {props.publication.user.firstName[0]}. {props.publication.user.lastName}
```
Comment (karmseveeratjisc) [inline ui/src/components/Publication/Card/index.tsx:29]: at some point would be nice to wrap this in a component, we have the same code repeated 7 or 8 times

File: ui/src/pages/author-guide.tsx [hunk 2]
```diff
@@ -80,8 +80,8 @@ const AuthorGuide: NextPage = (): React.ReactElement => (
                         <StandardText>
                             <ul className="ml-8 list-disc">
                                 <li className="mb-8">
-                                    <span className="font-bold">Sign in</span> using your ORCiD credentials (or create
-                                    an ORCiD if you don’t have one). You will also need to provide a verified email
+                                    <span className="font-bold">Sign in</span> using your ORCID® credentials (or create
+                                    an ORCID iD if you don’t have one). You will also need to provide a verified email
```
Comment (karmseveeratjisc) [inline ui/src/pages/author-guide.tsx:84]: wonder if this should say 

'create an ORCID account if you don't have one'

it was bad before, this doesn't really improve it..

Comment (florin-holhos) [reply ui/src/pages/author-guide.tsx:84]: Should I change it to "... create an ORCID account if you don't have one ..." ?

Comment (karmseveeratjisc) [reply ui/src/pages/author-guide.tsx:84]: yes

**Review by karmseveeratjisc — APPROVED** [review]

**Pull request merged**